qiskit/optimization/converters/ising_to_quadratic_program.py

```diff
@@ -134,24 +137,31 @@ def _create_qubo_matrix(self):
         # The other elements in the QUBO matrix is for quadratic terms of the qubit operator
         self._qubo_matrix = np.zeros((self._num_qubits, self._num_qubits))
 
-        for pauli in self._qubit_op.paulis:
+        if not isinstance(self._qubit_op, SummedOp):
```

Comment:
I am not sure I entirely follow the logic - summedOp is just one class of ListOps of type BaseOperator for which special treatment is applied here? Any thing else i.e. primitive ops and other list ops go via the alternative.

Reply:
Because calling `to_pauli_op()` on anything but a SummedOp returns a single PauliOp, only if called on a SummedOp it remains a SummedOp. E.g. `(PauliOp ^ CircuitOp).to_pauli_op() -> PauliOp` and `(PauliOp @ CircuitOp).to_pauli_op() -> PauliOp`. But `(PauliOp + CircuitOp).to_pauli_op() -> SummedOp(PauliOp, PauliOp)`.

Reply:
Is this true if the input is a ListOp (there the _to_pauli_op returns  a list of paulis as far as I know). So any ListOp type has oplist property. Maybe the latter cannot reasonably be called an Ising for this convertor.

Reply:
For a listop you would have to run the encoding for each element, this is something all converters could support. For now I added a type-check to disallow listops in the converter.